packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -712,16 +805,49 @@ class StackViewport extends Viewport implements IStackViewport {
   }
 
   private _resetProperties() {
-    let voiRange;
-    if (this._isCurrentImagePTPrescaled()) {
-      // if not set via setProperties; if it is a PT image and is already prescaled,
-      // use the default range for PT
-      voiRange = this._getDefaultPTPrescaledVOIRange();
+    const voiRange = this._getVOIRangeForCurrentImage();
+    this.setVOI(voiRange);
+
+    if (this.getRotation() !== 0) {
+      this.setRotation(0);
+    }
+    this.setInterpolationType(InterpolationType.LINEAR);
+    this.setInvertColor(false);
+  }
```

Comment:
There should not be any change for _resetProperties, why you are changing what it does? again the isCurrentImagePTPrescaled is needed, let's deal with that in another PR

Reply:
True, I forgot to put it back in here